web/src/test/java/se/inera/intyg/intygsadmin/web/controller/PrivatePractitionerControllerIT.java

```diff
@@ -66,4 +66,15 @@ public void testGetPrivatePractitionerFile() {
             .contentType(is(MediaType.APPLICATION_OCTET_STREAM.toString()));
     }
 
+    @Test
+    public void testUnregisterPrivatePractitionerOk() {
+        RestAssured.sessionId = getAuthSession(ADMIN_USER);
+
+        final String hsaId = "SE123456-X3";
+
+        given()
+            .when().delete(PRIVATE_PRACTITIONER_API_ENDPOINT + "/" + hsaId)
+            .then().statusCode(OK);
```

Comment:
Seems like we are missing a call to the method and verification :)

Reply:
Well it does make a call to the method and checks that the response is 200 (OK). Test fails if for example 404 (NOT_FOUND).